core/domain/prod_validation_jobs_one_off.py

```diff
@@ -3071,6 +3071,8 @@ def _validate_latex_strings_info(
                 html_validation_service.
                 get_latex_strings_without_svg_from_html(
                     html_strings_in_exploration))
+            decoded_latex_strings_without_svg = [
+                string.decode('utf-8') for string in latex_strings_without_svg]
```

Comment:
Did you test this manually by running the job on a local instance with unicode chars? I ask because I don't normally see this pattern -- it's not common for us to do .decode('utf-8'), so you may want to pay extra attention here.